internal/pkg/db/db_test.go

```diff
@@ -93,6 +106,114 @@ func (dbs *dbSuite) TestGetDocumentByID() {
+
+	// Normal (1 tag, 2 docs)
+	filteredDocs2, err := dbs.backend.FilterDocumentsByTag(docs, "tag2")
+	dbs.Require().NoError(err)
+	dbs.Require().Len(filteredDocs2, 2)
+	dbs.Require().Equal(filteredDocs2[0].GetMetadata().GetId(), dbs.documents[0].GetMetadata().GetId())
+	dbs.Require().Equal(filteredDocs2[1].GetMetadata().GetId(), dbs.documents[1].GetMetadata().GetId())
+
+	// Normal (another tag, 1 doc)
+	filteredDocs3, err := dbs.backend.FilterDocumentsByTag(docs, "tag3")
+	dbs.Require().NoError(err)
+	dbs.Require().Len(filteredDocs3, 1)
+	dbs.Require().Equal(filteredDocs3[0].GetMetadata().GetId(), dbs.documents[1].GetMetadata().GetId())
+
+	// Normal (multiple tags)
+	filteredDocs4, err := dbs.backend.FilterDocumentsByTag(docs, "tag1", "tag2", "tag3")
+	dbs.Require().NoError(err)
+	dbs.Require().Len(filteredDocs4, 2)
+	dbs.Require().Equal(filteredDocs4[0].GetMetadata().GetId(), dbs.documents[0].GetMetadata().GetId())
+	dbs.Require().Equal(filteredDocs4[1].GetMetadata().GetId(), dbs.documents[1].GetMetadata().GetId())
+
+	// Unknown tag
+	filteredDocs5, err := dbs.backend.FilterDocumentsByTag(docs, "unknown_tag")
+	dbs.Require().NoError(err)
+	dbs.Require().Empty(filteredDocs5)
+
+	// No tags
+	filteredDocs6, err := dbs.backend.FilterDocumentsByTag(docs)
+	dbs.Require().NoError(err)
+	dbs.Require().Len(filteredDocs6, 2)
+	dbs.Require().Equal(filteredDocs6[0].GetMetadata().GetId(), dbs.documents[0].GetMetadata().GetId())
+	dbs.Require().Equal(filteredDocs6[1].GetMetadata().GetId(), dbs.documents[1].GetMetadata().GetId())
+}
+
+func (dbs *dbSuite) TestSetAlias() {
+	docs, err := dbs.backend.GetDocumentsByID()
+	dbs.Require().NoError(err)
+
+	err = dbs.backend.RemoveAnnotations(docs[0].GetMetadata().GetId(), db.AliasAnnotation, "cdx")
+	dbs.Require().NoError(err)
+
+	// Error: Duplicate alias
+	err = dbs.backend.SetAlias(docs[0].GetMetadata().GetId(), "spdx", false)
+	dbs.Require().EqualError(err, "failed to set alias: alias already exists")
+
+	// Error: Duplicate alias
+	err = dbs.backend.SetAlias(docs[0].GetMetadata().GetId(), "spdx", true)
+	dbs.Require().EqualError(err, "failed to set alias: alias already exists")
+
+	// Set Alias (Normal)
+	err = dbs.backend.SetAlias(docs[0].GetMetadata().GetId(), "cdx", false)
+	dbs.Require().NoError(err)
+	docAlias, err := dbs.backend.GetDocumentUniqueAnnotation(docs[0].GetMetadata().GetId(), db.AliasAnnotation)
+	dbs.Require().NoError(err)
+	dbs.Require().Equal("cdx", docAlias)
+
+	// Error: Document already has an alias
+	err = dbs.backend.SetAlias(docs[0].GetMetadata().GetId(), "cdx2", false)
+	dbs.Require().EqualError(err, "the document already has an alias")
+
+	// Set Alias (Force)
+	err = dbs.backend.SetAlias(docs[0].GetMetadata().GetId(), "cdx2", true)
+	dbs.Require().NoError(err)
+	docAlias, err = dbs.backend.GetDocumentUniqueAnnotation(docs[0].GetMetadata().GetId(), db.AliasAnnotation)
+	dbs.Require().NoError(err)
+	dbs.Require().Equal("cdx2", docAlias)
+}
+
 func TestStoreSuite(t *testing.T) {
```

Comment:
Forgot to update this name when originally written

```suggestion
func TestDBSuite(t *testing.T) {
```